### PR TITLE
Route KeyStore logs to NativeLogger

### PR DESCRIPTION
**Pull request by akalin-keybase**
Also add a lot more KeyStore logs.

### CODE REVIEW

File: shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java [hunk 1]
```diff
@@ -62,91 +64,140 @@ private String keyStoreAlias(final String serviceName) {
     @SuppressLint("CommitPrefEdits")
     @Override
     public void clearSecret(final String serviceName, final String key) throws Exception {
-        prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
+        String id = serviceName + ":" + key;
+        NativeLogger.info("KeyStore: clearing secret for " + id);
+
+        try {
+            prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
```
Comment (akalin-keybase) [inline shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: here and below, I didn't actually change any code, just re-indented them for the try blocks

File: shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java [hunk 2]
```diff
@@ -62,91 +64,140 @@ private String keyStoreAlias(final String serviceName) {
     @SuppressLint("CommitPrefEdits")
     @Override
     public void clearSecret(final String serviceName, final String key) throws Exception {
-        prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
+        String id = serviceName + ":" + key;
+        NativeLogger.info("KeyStore: clearing secret for " + id);
```
Comment (chrisnojima) [inline shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: @patrickxb are these ids important at all? we can log them safely?

Comment (chrisnojima) [inline shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: vs the old code this is now logging a bunch of stuff, even in the happy path. i wonder if this'll hurt perf. we could just log the errors (the original goal)

Comment (patrickxb) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: They should be fine (should be `keybase:username`), but I just wanted to see the errors so if you want to drop these that's fine with me.

Comment (akalin-keybase) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: The secret store isn't in a hot path, and is consulted maybe a few times in a session, often only once at startup and if you log out/in. I don't think this'll hurt perf at all.

Comment (chrisnojima) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: i thought we used to his this a ton, i guess its cached now?

Comment (patrickxb) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: service caches retrievesecret.

Comment (chrisnojima) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: can i see a logsend w/ this in it

Comment (chrisnojima) [reply shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java]: chat me a link

**Pull request merged**